cargo-dist/src/backend/ci/github.rs

```diff
@@ -238,6 +322,60 @@ impl GithubCiInfo {
     }
 }
 
+fn vendor_repo(
+    root: &Utf8PathBuf,
+    nwo: &GithubRepoPair,
+    revision: &str,
+    tools: &Tools,
+) -> DistResult<()> {
+    let git = if let Some(tool) = &tools.git {
+        tool.cmd.to_owned()
+    } else {
+        return Err(DistError::ToolMissing {
+            tool: "git".to_owned(),
+        });
+    };
+
+    let url = format!("https://github.com/{nwo}.git");
+    let repo_path = root.join(&nwo.repo);
+    // If we're updating an existing vendored copy, remove it before
+    // cloning a new copy.
+    if repo_path.exists() {
+        std::fs::remove_dir_all(&repo_path)?;
```

Comment:
can we use axo asset instead?

Reply:
Didn't know it could do that! I'll use it.